Features/Context/PlatformUI.php

```diff
@@ -106,6 +102,22 @@ public function __construct($uri = self::PLATFORM_URI, $user = null, $password =
     /**
      * @BeforeScenario
      */
+    public function gatherContexts(BeforeScenarioScope $scope)
+    {
+        $refClass = new ReflectionClass($this);
+        $refProperties = $refClass->getProperties();
+        foreach ($refProperties as $refProperty) {
+            preg_match_all('#@(.*?)\n#s', $refProperty->getDocComment(), $matches);
+            $contexts = $this->parseAnnotations($matches[1]);
+            foreach ($contexts as $property => $context) {
+                $this->$property = $scope->getEnvironment()->getContext($context);
```

Comment:
If I read correctly the $context holds the full class Namespace to the context class.

It should be verified that this class implements a `Context`, otherwise this will try to assign *some* class instance to properties